elasticsearch-core/src/test/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClient6Test.scala

```diff
@@ -26,7 +27,7 @@ class RestlasticSearchClient6Test extends WordSpec with Matchers with BeforeAndA
   override val restClient = RestlasticSearchClient6Test.restClient
 
   "RestlasticSearchClient6" should {
-    behave like restlasticClient(restClient, IndexName, createIndex())
+    behave like restlasticClient(restClient, IndexName, createIndex(), TextType)
```

Comment:
How about creating a case class `Settings` and putting there `IndexName` and `TextType` to avoid long parameter list in the argument. Just an idea to improve.